CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 
+## [4.31.1](https://github.com/gravity-ui/page-constructor/compare/v4.31.0...v4.31.1) (2023-10-30)
```

Comment:
changelog is the generated by release robot, you don't need to do it manually

Reply:
I did not. It was my messy rebase main branch. fixed